modules/apps/dynamic-data-mapping/dynamic-data-mapping-api/src/main/java/com/liferay/dynamic/data/mapping/util/DDMFormFieldTemplateContextContributorUtil.java

```diff
@@ -22,6 +26,28 @@
  */
 public class DDMFormFieldTemplateContextContributorUtil {
 
+	public static Map<Locale, String> getListTypeEntryNameMap(
+		DDMFormField ddmFormField,
+		ListTypeEntryLocalService listTypeEntryLocalService, String key) {
```

Comment:
Sort it please: `k` first, then `l` :+1:

Reply:
Done!